code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -136,7 +136,7 @@
 	return TRUE
 
 /datum/reagent/consumable/cooking_oil/reaction_turf(turf/open/T, reac_volume)
-	if(!istype(T))
+	if(!istype(T) || istype(T, /turf/open/space))
```

Comment:
isspaceturf(T)

should probably also check lava & chasms too.

Reply:
@ShizCalev do we have a list of turfs like these

Reply:
i added one